### DISCUSSION TITLE
Question about operators

### DISCUSSION THREAD

**Discussion by merlinsun** [cvc5/cvc5#8378] [Q&A] [answered]
Hi!
I have a question about the operators that are not documented in the smtlib standard, such as `sin`, `cos`, and `exp`.
How can I know all those operators supported by cvc5?

Best

**Comment by nafur** [comment, marked as answer]
Hi!
Transcendental (or trigonometric) functions are not part of the SMT-LIB standard, but a custom extension of cvc5 (though it should coincide with what some other solvers are doing). We generally include documentation on non-standard theories at https://cvc5.github.io/docs/cvc5-0.0.7/theories/. For these in particular, this would be the "transcendentals".
Is this what you are looking for?
Best, Gereon

**Comment by merlinsun** [comment]
Thanks, Gereon!
That's what I want to know.
